example_step_workflow/steps/fancyplot/fancyplot.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import logging
+from pathlib import Path
+from typing import List, Optional, Union
+
+import matplotlib
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+from datastep import Step, log_run_params
+from tqdm import tqdm
+
+from ..sum import Sum
+
+from example_step_workflow.steps.fancyplot.plot_utils import gradient_fill
+
+matplotlib.use("agg")
+plt.style.use("seaborn-whitegrid")
+
+###############################################################################
+
+log = logging.getLogger(__name__)
+
+
+###############################################################################
+
+
+class Fancyplot(Step):
+    def __init__(
+        self,
+        direct_upstream_tasks: List["Step"] = [Sum],
+        config: Optional[Union[str, Path]] = None,
+    ):
+        super().__init__(direct_upstream_tasks=direct_upstream_tasks, config=config)
+
+    @log_run_params
+    def run(
+        self,
+        vectors: Optional[Union[Union[str, Path], List[Path]]] = None,
+        filepath_column: str = "filepath",
+        **kwargs,
+    ) -> List[Path]:
+        """
+        Run a pure function.
+
+        Protected Parameters
+        --------------------
+        distributed_executor_address: Optional[str]
+            An optional executor address to pass to some computation engine.
+        clean: bool
+            Should the local staging directory be cleaned prior to this run.
+            Default: False (Do not clean)
+        debug: bool
+            A debug flag for the developer to use to manipulate how much data runs,
+            how it is processed, etc.
+            Default: False (Do not debug)
```

Comment:
These can simply be removed since you aren't using them.

Reply:
Done!